packages/orbit-components/src/primitives/MobileDialogPrimitive/index.js

```diff
@@ -41,10 +41,16 @@ const MobileDialog = ({
     [clearRenderTimeout, setRender, setshownWithTimeout, stopPropagation],
   );
 
-  const handleOutMobile = useCallback(() => {
-    setshown(false);
-    setRenderWithTimeout(false);
-  }, [setRenderWithTimeout, setshown]);
+  const handleOutMobile = useCallback(
+    ev => {
+      if (stopPropagation) {
+        ev.stopPropagation();
+      }
+      setshown(false);
+      setRenderWithTimeout(false);
+    },
+    [setRenderWithTimeout, setshown],
```

Comment:
ESLint correctly warns about this:
```suggestion
    [setRenderWithTimeout, setshown, stopPropagation],
```

Reply:
Thanks for catching this! ESLint was not cooperating with me well today so this slipped 😅